lua/elixir/utils.lua

```diff
@@ -19,13 +19,7 @@ function M.root_dir(fname)
   local child_or_root_path =
     vim.fs.dirname(vim.fs.find({ "mix.exs", ".git" }, { upward = true, path = fname })[1])
```

Comment:
I recently learned that `vim.fs.find` can take a `limit` option, that we can set to `2`.

I think this might be sufficient and easier than the current state of things.

So it'd be like 

```lua
local matches = vim.fs.find({"mix.exs"}, {upward = true, path = fname, limit = 2})

return matches[2] or matches[1]
```

Can you try that out? I think we can also drop the `vim.loop.os_homedir()` part, but i'm not sure.

I'd also like to get a test written for this, but if you can confirm that it works by manually testing it, I can contribute the test. i think it would need some test fixtures.

Reply:
> Can you try that out?

Yes, I'll try that.

> I think we can also drop the `vim.loop.os_homedir()` part, but i'm not sure.

Indeed, maybe a warning with `vim.notify` could be used if no path is found.

> I'd also like to get a test written for this, but if you can confirm that it works by manually testing it, I can contribute the test. i think it would need some test fixtures.

I can try writing those.

Reply:
Awesome, really appreciate it!

Reply:
I've added a dir `fixtures` with two elixir "projects". I never wrote tests for neovim before so I don't we if there's a better way to add fixtures to tests, at least I couldn't find it.